### DISCUSSION TITLE
Request for information on "Integrated DR" and "zero downtime" functionality coming soon

### DISCUSSION THREAD

**Discussion by prlcutting** [datalust/seq-tickets#1278] [Help] [answered]
Hi.  The [Seq pricing page](https://datalust.co/pricing) teases some new integrated Disaster Recovery (DR) functionality coming soon for Enterprise plans and above.  Searching around I also see hints of [High Availability (HA)](https://github.com/datalust/seq-tickets/discussions/1103) and [zero downtime](https://blog.datalust.co/seq-2021-3-previews-now-available/) being mentioned too.

Can you point me to any information you have on what we can expect please (website, blog post etc.) so that we can learn more, or perhaps share some more detailed information here?  Our Team-level license is up for renewal soon, and we're considering upgrading to the Enterprise plan for the DR functionality, but would like to know more about it first.  Thanks.

It looks like the new functionality will be released in 2021.3 - is that Q3 of 2021, i.e. very soon?  Thanks again.

**Comment by nblumhardt** [comment, marked as answer]
Hi! Thanks for getting in touch.

The current best source of information is:

https://docs.datalust.co/v2021.3/docs/setting-up-a-disaster-recovery-dr-instance

It would be great to know if there's any crucial information missing there, so we can improve the docs.

Regarding the timeline, we're working on a "ready-when-it's-ready" schedule, but will ideally publish a feature-complete 2021.3 preview this month, and will be working to stabilize it for a final release from there on. Historically the stabilization period from feature-complete to final release can be anywhere from a few weeks to a month or two. If not Q3, then we'd expect to land it fairly early in Q4 🤞 .

Hope this helps, and if you need more info or would like to spend some time walking through DR with us via Zoom etc. please just let me know!

Nick

**Reply by prlcutting** [reply]
This is exactly the information I was looking for.  Thank you for the quick response, and for the offer of assistance.  We might take you up on that later!  Thanks again.
